shared-module/bitmaptools/__init__.c

```diff
@@ -918,3 +921,64 @@ void common_hal_bitmaptools_alphablend(displayio_bitmap_t *dest, displayio_bitma
         }
     }
 }
+
+STATIC void draw_circle(displayio_bitmap_t *destination,
+    int16_t x, int16_t y,
+    int16_t radius, uint32_t value) {
+
+    int16_t d, yb;
+
+    mp_arg_validate_int_range(x0, SHRT_MIN, SHRT_MAX, MP_QSTR_x);
+    mp_arg_validate_int_range(y0, SHRT_MIN, SHRT_MAX, MP_QSTR_y);
```

Comment:
Missed these renames.
```suggestion
    mp_arg_validate_int_range(x, SHRT_MIN, SHRT_MAX, MP_QSTR_x);
    mp_arg_validate_int_range(y, SHRT_MIN, SHRT_MAX, MP_QSTR_y);
```